### PR TITLE
improved docs

### PR DESCRIPTION
**Pull request by kanolato**
I think it would be nice to have some docs of persistence

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
       helm list
     NAME                REVISION    UPDATED                     STATUS      CHART                   NAMESPACE
     my-confluent-oss    1           Tue Jun 12 16:56:39 2018    DEPLOYED    cp-helm-charts-0.1.0    default 
+Persistence
+~~~~~~~~~~~~~~~~~~~
+The ZooKeeper and Kafka cluster are deployed with ``StatefulSets`` that have a ``volumeClaimTemplate`` which will provide the persistent volume for each replica. We can define the size of the volumes by changing ``dataDirSize`` and ``dataLogDirSize`` under ``cp-zookeeper`` and ``size`` under  ``cp-kafka`` in `values.yaml <https://github.com/confluentinc/cp-helm-charts/blob/master/values.yaml>`__
```
Comment (joel-hamill) [inline docs/index.rst]: suggest 

```
...that have a ``volumeClaimTemplate`` which provides...
```

Comment (joel-hamill) [inline docs/index.rst]: suggest (see [Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone)):

```

You can define the size of the volumes by changing...
```

File: docs/index.rst [hunk 2]
```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
...
+
+We also can use `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/>`__ typically to use the cloud provider's volumes. For example if we are on AWS our storage class will look like this:
```
Comment (joel-hamill) [inline docs/index.rst]: suggest:

```
You also use the cloud provider's volumes by specifying `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/>`__.
```

Comment (joel-hamill) [inline docs/index.rst]: suggest:

```
For example, if you are on AWS your storage class will look like this:
```

File: docs/index.rst [hunk 3]
```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
...
+
+      apiVersion: storage.k8s.io/v1beta1
+      kind: StorageClass
+      metadata:
+        name: ssd
+      provisioner: kubernetes.io/aws-ebs
+      parameters:
+        type: gp2
+
+.. tip:: Read the Kubernetes `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters>`__ documentation to adapt this example to your needs.
```
Comment (joel-hamill) [inline docs/index.rst]: suggest: 

```
.. tip:: To adapt this example to your needs, read the Kubernetes `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters>`__ documentation.
```

File: docs/index.rst [hunk 4]
```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
...
+
+The ``StorageClass`` that was created can be specified in ``dataLogDirStorageClass`` and ``dataDirStorageClass`` under 
+``cp-zookeeper`` and in ``storageClass`` under ``cp-kafka`` in `values.yaml <https://github.com/confluentinc/cp-helm-charts/blob/master/values.yaml>`__.
+
+Non persistent Kafka and ZooKeeper cluster's can also be deployed, you just have to change the value of ``persistence.enabled`` under ``cp-kafka`` and ``cp-zookeeper`` in `values.yaml <https://github.com/confluentinc/cp-helm-charts/blob/master/values.yaml>`__ . This kind of clusters are suitable for development and testing purposes. The ``StatefulSets`` are going to use ``emptyDir`` volumes, this means that its content is strictly related to the pod life cycle and is deleted when the pod goes down.
```
Comment (joel-hamill) [inline docs/index.rst]:
```
To deploy non-persistent Kafka and ZooKeeper clusters, you must change the value of...
```

Comment (joel-hamill) [inline docs/index.rst]: suggest

```
These type of clusters are suitable for...
```

File: docs/index.rst [hunk 5]
```diff
@@ -233,6 +255,8 @@ Scaling
 
 .. tip:: All scaling operations should be done offline with no producer or consumer connection.
 
+.. tip:: The number of nodes should always be odd number
```
Comment (joel-hamill) [inline docs/index.rst]: add a period

**Comment by kanolato** [comment]
I could not see the [style guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone)
_Your email address miguel@circulosiete.com doesn't have access to confluentinc.atlassian.net_.
Thanks for the review!, i just made a commit with the changes.

File: docs/index.rst [hunk 6]
```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
       helm list
     NAME                REVISION    UPDATED                     STATUS      CHART                   NAMESPACE
     my-confluent-oss    1           Tue Jun 12 16:56:39 2018    DEPLOYED    cp-helm-charts-0.1.0    default 
+Persistence
```
Comment (gAmUssA) [inline docs/index.rst]: add new line between sections

File: docs/index.rst [hunk 7]
```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
...
+      kind: StorageClass
+      metadata:
+        name: ssd
+      provisioner: kubernetes.io/aws-ebs
+      parameters:
+        type: gp2
+
+.. tip:: To adapt this example to your needs, read the Kubernetes `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters>`__ documentation.
+
+The ``StorageClass`` that was created can be specified in ``dataLogDirStorageClass`` and ``dataDirStorageClass`` under
```
Comment (gAmUssA) [inline docs/index.rst]: don't break line.
modern editors support word wrapping.
also this https://github.com/brandon-rhodes/blog/blob/master/texts/brandon/2012/one-sentence-per-line.rst

**Comment by kanolato** [comment]
Thanks for the comments! i just made a commit with the changes.

**Comment by gAmUssA** [comment]
@kanolato thanks for the help!

**Pull request merged**